doc/whats_new.rst

```diff
@@ -276,6 +276,11 @@ Bug fixes
    - Fixed a bug in :class:`svm.OneClassSVM` where it returned floats instead of
      integer classes. :issue:`8676` by :user:`Vathsala Achar <VathsalaAchar>`.
 
+   - Fix a bug where :func:`sklearn.tree.export_graphviz` will raise an error
+     when the length features_names does not match n_features in the decision
+     tree.
+     :issue:`8512` by `Li Li <https://github.com/aikinogard>`_.
```

Comment:
Can you use this syntax? https://github.com/scikit-learn/scikit-learn/pull/8512/files#diff-3433b590ca8611f80d7687b51a869383R277